pkg/synthetictests/duplicated_events.go

```diff
@@ -191,3 +214,85 @@ func getTimesAnEventHappened(message string) (string, int) {
 	}
 	return matches[0][1], int(times)
 }
+
+// isConsoleReadinessDuringInstallation returns true if the event is for console readiness and it happens during the
+// initial installation of the cluster.
+// we're looking for something like
+// > ns/openshift-console pod/console-7c6f797fd9-5m94j node/ip-10-0-158-106.us-west-2.compute.internal - reason/ProbeError Readiness probe error: Get "https://10.129.0.49:8443/health": dial tcp 10.129.0.49:8443: connect: connection refused
+// with a firstTimestamp before the cluster completed the initial installation
+func isConsoleReadinessDuringInstallation(monitorEvent monitorapi.EventInterval, kubeClientConfig *rest.Config) bool {
+	if !strings.Contains(monitorEvent.Locator, "ns/openshift-console") {
+		return false
+	}
+	if !strings.HasPrefix(monitorEvent.Locator, "pod/console-") {
```

Comment:
your example a few lines up in the godocs opens with `ns/openshift-console `, so would expect `Contains` instead of `HasPrefix` here.